### PR TITLE
(fix)[GRO-1409] move NFTs from home nav to artwork medium

### PR DESCRIPTION
**Pull request by TMcMeans**
The type of this PR is: fix

This PR solves [GRO-1409]

### Description

The goal of this PR is to clean up the nav bar a bit as part of the fix it sprint. NFTs now live under artwork mediums where they belong :)

Desktop

https://user-images.githubusercontent.com/23108927/203132713-f8d7cae2-4cc5-4ff6-af5b-cae4bb7bb376.mp4

MWEB

https://user-images.githubusercontent.com/23108927/203132974-ad27c625-f9cd-489f-a2f9-e77fbf4b525e.mp4

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[GRO-1409]: https://artsyproduct.atlassian.net/browse/GRO-1409?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dzucconi — APPROVED** [review]
🫡

File: src/Components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -422,15 +422,6 @@ export const NavBar: React.FC = track(
...
                 </NavBarItemLink>
-
-                <NavBarItemLink
-                  href="/collect?additional_gene_ids[0]=nft"
-                  onClick={handleClick}
-                  data-label="NFTs"
-                  display={["none", "none", "flex"]}
-                >
-                  {t`navbar.nfts`}
-                </NavBarItemLink>
```
Comment (damassi) [inline src/Components/NavBar/NavBar.tsx:433]: 👋

Comment (TMcMeans) [reply src/Components/NavBar/NavBar.tsx:433]: that was removed, was it in the wrong place??

Comment (dzucconi) [reply src/Components/NavBar/NavBar.tsx:433]: I think he was waving goodbye :p

Comment (TMcMeans) [reply src/Components/NavBar/NavBar.tsx:433]: oh haha yea adios, NFTs!

**Pull request merged**